ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -707,9 +706,9 @@ export class AccountCreation extends React.Component<
                     {this.state.countryDropdownSelection === Country.OTHER && (
                       <div style={{ marginTop: '0.3rem' }}>
                         <TextInput
-                          id='country'
-                          name='country'
-                          aria-label='Country input'
+                          id='other country'
```

Comment:
most countries are in the dropdown.  This text area is reserved for only when the user chooses OTHER